src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1122,7 +1115,7 @@ func main() {
 	logging.SetLogLevel("autorelay", "info") // Logs relayed byte counts spammily
 	logging.SetLogLevel("providers", "debug")
 	logging.SetLogLevel("dht/RtRefreshManager", "warn") // Ping logs are spammy at debug, cpl logs are spammy at info
-	logging.SetLogLevel("dht", "info") // Logs every operation to debug
+	logging.SetLogLevel("dht", "debug")                 // Logs every operation to debug
```

Comment:
Will we want to leave these logs enabled, or is this just for testing? It's nice to not have the logspam if we aren't going to be using them.